crates/apps/src/lib/cli.rs

```diff
@@ -4618,16 +4705,21 @@ pub mod args {
     impl CliToSdk<UpdateStewardCommission<SdkTypes>>
         for UpdateStewardCommission<CliTypes>
     {
+        type Error = std::io::Error;
+
         fn to_sdk(
             self,
             ctx: &mut Context,
-        ) -> UpdateStewardCommission<SdkTypes> {
-            UpdateStewardCommission::<SdkTypes> {
-                tx: self.tx.to_sdk(ctx),
+        ) -> Result<UpdateStewardCommission<SdkTypes>, Self::Error> {
+            let tx = self.tx.to_sdk(ctx)?;
+            let _commission = std::fs::read(self.commission)?;
```

Comment:
```suggestion
            let commission = std::fs::read(self.commission)?;
```
nit: why did you use an underscore for the identifier here?